commit b52495e21f07114b3800a861cb24107ae8f0a2e3 (refs/changes/36/736/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-19T11:20:20-07:00 (3 years ago)
    
    IMGAPI-596 update imgapi and docker to use restify-clients@1.4.0 with upstreamed http_proxy support

diff --git a/README.md b/README.md
index 2f769b1..1885c22 100644
--- a/README.md
+++ b/README.md
@@ -23,6 +23,8 @@ Node.js client libraries for other Triton APIs are in other repos. For example s
 - [node-moray](https://github.com/node-moray): client for Triton's core Moray service
 - [node-triton](https://github.com/node-triton): Triton CloudAPI client and CLI
 - [node-manta](https://github.com/node-manta): Manta client and CLI
+- [wf-client](https://github.com/sdc-wf-client): client for Triton's core
+  Workflow API service
 
 
 ## Testing
diff --git a/package.json b/package.json
index 09d5076..148ef53 100644
--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "lru-cache": "2.3.0",
     "libuuid": "0.2.1",
     "once": "^1.3.1",
-    "restify-clients": "git+https://github.com/trentm/clients.git#0b989e0bbd375ddeefce2f6e2754f66e5dbb59d1",
+    "restify-clients": "^1.4.0",
     "restify-errors": "^3.0.0",
     "vasync": "^1.6.2",
     "verror": "^1.6.0",
